cocotb/share/makefiles/Makefile.sim

```diff
@@ -30,8 +30,71 @@
 # This is a simple wrapper Makefile to pull in the appropriate Makefile for
 # the desired simulator and set up common paths for the sims to use to build with
 
+.PHONY: all
 all: sim
 
+# NOTE: make sure to keep "helpmsg" aligned with documentation/source/building.rst
+# Also keep it at 80 chars.
```

Comment:
Keeping this aligned sounds like a lot of effort - can we just bundle this RST file with the install, and read it in here?

Reply:
"Aligned" does not mean "duplicate", the text in ``helpmsg`` is shortened to be the essence of the description for people who want to refresh their memory and may leave out details.
I do not expect this to change often anymore, after all, we want to get rid of Makefiles at some point. Once we have them replaced with Python scripts, we can use ``sphinx-argparse`` to get the argument descriptions into the documentation.